### DISCUSSION TITLE
Update on a ready to use windows build

### DISCUSSION THREAD

**Discussion by Pamapama** [FreeRDP/FreeRDP#6770] [General]
Hello,

I was wondering if anybody could build a current version for windows users. 
After using mstsc.exe for too long I would like to switch to freerdp, but I cant build it myself.
The only version I found is an old 1.x version, which can't do the dynamic resolution.

Edit: i found the daily build page with a current version. Unfortunately dynamic resolution is not working when connecting from windows 10 to another windows 10 machine.

Thanks a lot,
Patrick

**Comment by akallabeth** [comment]
1. Go to https://www.freerdp.com/
2. Press `Download now`, be redirected to https://github.com/FreeRDP/FreeRDP/wiki/PreBuilds
3. Download the desired binary.

Just a warning, `dynamic-resolution` is currently not supported by the windows client.
